src/app/core/entity-components/entity-list/EntityListConfig.ts

```diff
@@ -37,6 +38,12 @@ export interface EntityListConfig {
    */
   filters?: FilterConfig[];
 
+  /**
+   * Optional initial sort order.
+   * Default is to sort by the first column.
+   */
+  sortByDefault?: Sort;
```

Comment:
I would understand this as a question: "Sort By default? Yes/No". How about `defaultSort`?

Reply:
fair point :-)